### DISCUSSION TITLE
Question when setting primary to false

### DISCUSSION THREAD

**Discussion by Emzi-beta** [ika-rwth-aachen/mqtt_client#55] [General]
If I false primary, mqtt's topic is designated mqtt_client/ros_msg_type/{topic name} and I think I should get it through mqtt_sub -h 127.0.0.1 -t /mqtt_client/ros_msg_type/{topic name} (mqtt sub command using CLI), but it's received as mqtt_sub -h 127.0.0.1 -t /{topic name}. Is it actually publish as /{topic name}?

If primary = false, can I specify the type?

**Comment by lreiher** [comment]
Sorry, I seem to have overlooked your question. Can you clarify, I don't really understand the question?
